### ISSUE TITLE
Warning sending color to searchBar

### ISSUE THREAD

**Issue by martincarrera** [react-native-training/react-native-elements#320]
### Description

When trying to send a style with the param `color` on the `SearchBar` `inputStyle`, it shows a warning with the following information:

```
Warning: Failed prop type: Invalid props.inputStyle key `color` supplied to `Search`.
Bad object: {
  "color": "#2f95dc",
  "backgroundColor": "#fff"
}
Valid keys: [
  "width",
  "height",
  "top",
  "left",
  "right",
  "bottom",
  "minWidth",
  "maxWidth",
  "minHeight",
  "maxHeight",
  "margin",
  "marginVertical",
  "marginHorizontal",
  "marginTop",
  "marginBottom",
  "marginLeft",
  "marginRight",
  "padding",
  "paddingVertical",
  "paddingHorizontal",
  "paddingTop",
  "paddingBottom",
  "paddingLeft",
  "paddingRight",
  "borderWidth",
  "borderTopWidth",
  "borderRightWidth",
  "borderBottomWidth",
  "borderLeftWidth",
  "position",
  "flexDirection",
  "flexWrap",
  "justifyContent",
  "alignItems",
  "alignSelf",
  "overflow",
  "flex",
  "flexGrow",
  "flexShrink",
  "flexBasis",
  "aspectRatio",
  "zIndex",
  "shadowColor",
  "shadowOffset",
  "shadowOpacity",
  "shadowRadius",
  "transform",
  "transformMatrix",
  "decomposedMatrix",
  "scaleX",
  "scaleY",
  "rotation",
  "translateX",
  "translateY",
  "backfaceVisibility",
  "backgroundColor",
  "borderColor",
  "borderTopColor",
  "borderRightColor",
  "borderBottomColor",
  "borderLeftColor",
  "borderRadius",
  "borderTopLeftRadius",
  "borderTopRightRadius",
  "borderBottomLeftRadius",
  "borderBottomRightRadius",
  "borderStyle",
  "opacity",
  "elevation"
]
```

Sample SearchBar:

```
<SearchBar
    inputStyle={{color: '#CCCCCC'}}
/>
```

Even though it works, the `SearchBar` component should not throw a warning.

### Reproduction Steps and Sample Code

It was not reproducible on snack.expo.io. [Sample code](https://snack.expo.io/ByK-J7ERe)

It is reproducible when developing an application. 

Steps to reproduce:

1. Create a new Expo App using tabs.
2. Replace the `app/screens/HomeScreen.js` file with the following content:

```
import React from 'react'
import {
  StyleSheet,
  View,
} from 'react-native'

import Colors from '../constants/Colors'

import { SearchBar } from 'react-native-elements'

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
})

export default class HomeScreen extends React.Component {
  static route = {
    navigationBar: {
      title: 'HomeScreen',
      backgroundColor: Colors.tintColor,
      borderBottomWidth: 0,
      tintColor: '#fff',
    },
  }

  render() {
    return (
      <View style={{flex: 1, backgroundColor: '#fff'}}>
        <SearchBar
          inputStyle={{color: '#CCCCCC'}}
        />
      </View>
    )
  }
}
```

3. Run the app on a simulator (iOS or android) or a device.

4. The warning appears

<p align="center">
  <img width="300px" src="https://puu.sh/voKvb/5f629034f6.png""/>
  <img width="300px" src="https://puu.sh/voKQh/49d38da92a.png""/>
</p>

**Comment by binoy14** [comment]
@martincarrera first of all thank you so much for such a comprehensive details for this issue! This is a bug and will be fixed in the next release

**Comment by martincarrera** [comment]
Thank you! And thanks for this awesome project, I'll be waiting for the next release then.

**Comment by bozzmob** [comment]
Hi @martincarrera , I checked both your sample [code](https://snack.expo.io/ByK-J7ERe). Also added the default style argument before the color props, and that works fine too. Here is my [sample](https://snack.expo.io/H1nXUeBAe).

Tested both on Expo and Samsung Galaxy S7 Edge.

Please let me know how you reproduce the issue. I don't see the warning.

**Comment by martincarrera** [comment]
Hi @bozzmob, thanks for your reply. As I said previously, it is NOT reproducible on Expo Snack

Did you follow the `Steps to reproduce`? Or did you just scan the QR of the Expo Snack?

It happens on iOS simulator, Android simulator, and device (Samsung Galaxy S7 Edge, and iPhone 7)

Let me know if you still can't reproduce this issue.

**Comment by binoy14** [comment]
Closing this in favor of #323 thanks for reporting

**Comment by bozzmob** [comment]
This has been fixed and will be available from v0.11.2

**Comment by martincarrera** [comment]
Thanks @bozzmob!
